src/keys.ts

```diff
@@ -0,0 +1,17 @@
+import { purry } from './purry';
+
+function _keys<T>(object: { [key: string]: T } | ArrayLike<T>) {
+  return Object.keys(object) as string[];
+}
+
+// data-first
+export function keys<T>(object: { [key: string]: T } | ArrayLike<T>): string[];
```

Comment:
please add js-doc documentation to the functions, because the remeda docs will be auto-created from it. Please have a look at other functions for inspiration.